### ISSUE TITLE
Mark One: Error Message Color Contrast

### ISSUE THREAD

**Issue by jonseitz** [seas-computing/course-planner#598]
The Text in our Error Message does not have sufficient Color Contrast. We should change the background color to our "Negative Dark" variant instead of Negative Medium.

**Comment by jonseitz** [comment]
Closed by seas-computing/mark-one#140
